torch/onnx/_internal/fx/fx_exporter.py

```diff
@@ -1,23 +1,295 @@
+from __future__ import annotations
+
 import abc
-from typing import Any, Callable, Mapping, Sequence
+import inspect
+from typing import Any, Callable, Mapping, Optional, Sequence, Tuple, Type
 
 import torch._ops
 import torch.fx
 
 import torch.onnx
-import torch.onnx._internal.exporter
 import torch.onnx._internal.fx.function_dispatcher as function_dispatcher
 import torch.onnx._internal.fx.passes as passes
-from torch.onnx._internal import _beartype
+from torch.onnx._internal import _beartype, exporter
+from torch.onnx._internal.fx import fx_exporter
```

Comment:
Is this file importing itself?

Reply:
great catch, how is this not failing ..

Reply:
`mypy` is letting me down too, even after `make setup_lint` and `lintrunner init` and finally `lintrunner`. CI finds a bunch of failures while local runs pass